### ISSUE TITLE
Create new group in QA environment for MST Courtesy Cards

### ISSUE THREAD

**Issue by angela-tran** [cal-itp/benefits#2574]
From the discussion in https://cal-itp.slack.com/archives/C022HHSEE3F/p1733508345620659:

> I'd like to create brand new group for Courtesy Cards in Littlepay QA so we have a clean / no expiration group to test with. And then we have to be strict and not enable expiration against this group for testing.

TLDR; at some point we enrolled cards into the existing group with expiration dates, and now we see a `NotImplementedError` when trying to test enrollment with those cards. This change will get us back in a clean state.

## Acceptance Criteria

- [ ] New group is created
- [ ] Group ID is configured for MST Courtesy Card flow in `dev` and `test`

**Comment by lalver1** [comment]
- [x] Confirmed that the existing `Courtesy Card discount group` had funding sources with expiration dates
- [ ] Created a brand new `Courtesy Card discount group` in Littlepay MST QA
- [ ] Configured the `Courtesy Card` enrollment flow in `dev` and `test` with the group ID of the new `Courtesy Card discount group`

**Comment by thekaveman** [comment]
@lalver1 

> Updated the template names of the Courtesy Card enrollment flow in dev as prompted by the validation error

Can you say more about this? I thought we were just using the default (calculated) templates for each of these?

**Comment by lalver1** [comment]
Sure thing @thekaveman , I wasn't expecting this validation error, but when I updated the value in the `Group id` field for the `Courtesy Card` enrollment flow in `dev` and tried to save it, I got the error. The 3 templates in the screenshot were renamed to `selection-label--mst-agency-card.html`, `start--mst-agency-card.html`, and `success--mst-agency-card.html` at some point after `2024.11.3-rc1`. I think we never saw the error because `dev` only has CST as the active agency, so Django never went looking for these renamed templates. But now I'm thinking about the calculated templates feature and trying to make sense of why I got that validation error.

On `test` I did not get the validation error, and it makes sense because when I check out the `2024.11.3-rc1` tag (b69e58cd6101853d30f399b7aed4dfa941442c8f), I see that these 3 templates are still named `selection-label--mst-courtesy-card.html`, `start--mst-courtesy-card.html`, and `success--mst-courtesy-card.html`, so that's the source that is on `test`.

**Comment by thekaveman** [comment]
Thanks @lalver1. That's right, we don't use `courtesy-card` anymore in the template names, instead using the generic `agency-card`. So those values must have been the old values that were in the configuration DB from before #2471?

So to be clear: are we now _configuring_ the `agency-card` template name, or are we using the _calculated_ `ageny-card` template name?

**Comment by lalver1** [comment]
Yep, those were the old values that were in the configuration DB from before #2471.

So that's a good point @thekaveman, I don't need to configure these 3 fields since we don't need to override anything, I should just leave them blank. I'll make that change in `dev` and thanks for helping me remember about this change.
